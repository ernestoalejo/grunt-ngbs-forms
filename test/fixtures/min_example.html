
:before:
<%= buildForm('test/fixtures/min_example.form') %>
:after:
