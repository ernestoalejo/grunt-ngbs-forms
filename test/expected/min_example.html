
:before:
<form role="form" name="f0" novalidate ng-init="f0.sent = false;" ng-submit="f0.sent = true; f0.$setPristine(); f0.$valid && submit();" autocomplete="off">
<fieldset>

  <div class="form-group">
    
    <label for="f0foo" class="control-label">My label</label>

    <div>
      <input type="text" placeholder="" class="form-control" name="f0foo" id="f0foo" ng-model="data.foo">
    </div>

  </div>

</fieldset>
</form>

:after:
